packages/office-ui-fabric-react/src/components/Dialog/DialogContent.base.tsx

```diff
@@ -113,7 +112,7 @@ export class DialogContentBase extends React.Component<IDialogContentProps, {}>
       contents: [],
     };
 
-    React.Children.map(this.props.children, child => {
+    React.Children.map(this.props.children, (child) => {
```

Comment:
This looks like it might have been caused by mismatched prettier versions--should probably be reverted to avoid possibly failing the build (if we have that check set up in v7, I forget)
```suggestion
    React.Children.map(this.props.children, child => {
```